app/models/date_range.rb

```diff
@@ -0,0 +1,33 @@
+class DateRange
+  
+  # Step 1: only in use with Conference date range.
+  # Validated in Conference: presence and chronological order
+  #  because the input is done there.
+  
+  def initialize(start_date:, end_date:)
+    @start_date = start_date
+    @end_date = end_date
+  end
+  
+  # Accepts dates
+  # Returns a string with one or more dates, with
+  # the following formats:
+  # 12 - 13 Jul 2017 or 30 Jul - 1 Aug 2017 or
+  # 31 Dec 2017 - 5 Jan 2018 or 1 Dec 2017 - 12 Dec 2018
+  def display
```

Comment:
May I suggest naming this method `#to_s`? That way a `DateRange` instance can be used in `"string #{interpolations}"` and it follows a common interface amongst Ruby object. Reusing method names for similar functionality is very important in terms of polymorphism (Duck Typing in Ruby's case).